server/packaging/module_skeleton/lib/puppet/face/language_server/quickstart.rb

```diff
@@ -0,0 +1,32 @@
+require 'puppet/face'
+
+Puppet::Face.define(:language_server, '0.1.0') do
+  action :quickstart do
+    summary "Start a Puppet Language Server with quick defaults"
+    description <<-EOT
+      Starts a puppet language server with quick default settings:
+        - Port 8080
+        - No connection timeout
+        - Listen on all interfaces
```

Comment:
This also conflicts with `start.rb` - I'm now beginning to believe I've fundamentally missed or misunderstood something, because looking further the timeout also doesn't match. Can you help me grok this?